WordPress/Classes/Extensions/Math.swift

```diff
@@ -22,14 +22,13 @@ extension Int {
 }
 
 extension Comparable {
-    /**
-     Clamps self between a minimum and maximum value
-     
-     - returns: the method returns
-        - min if self < min
-        - max if self > max
-        - otherwise it returns self
-     */
+    /// Clamps self between a minimum and maximum value
+    ///
+    /// - Returns: the method returns
+    ///     - min if self < min
+    ///     - max if self > max
+    ///     - otherwise it returns self
+    ///
```

Comment:
I'm divided about that blank line before the method, looking at the [stdlib](https://github.com/apple/swift/blob/master/stdlib/public/core/Collection.swift) again it's not there. I agree that sometimes it helps adding some visual space, although in cases where the documentation is one line it seems like a lot:

```swift
/// The MediaType for the asset
///
var assetMediaType: MediaType { get }
```

For now, I think I'd rather have consistency so I'm fine with either, but something to discuss when working on the style guide.